backend/src/server.spec.ts

```diff
@@ -15,21 +15,24 @@ const finishedAsync:any = promisify(finished);
 
 describe('server.ts - Express application', () => {
   let totalPersons: number;
+  const apiPath = (api: string): string => {
+    return `${process.env.BACKEND_PROXY_PATH}/api`
```

Comment:
Otherwise it won't use the input parameter
```suggestion
    return `${process.env.BACKEND_PROXY_PATH}/${api}`
```